### PR TITLE
Update the build scripts to use the Makefile.

### PR DESCRIPTION
**Pull request by mpark**

### CODE REVIEW

File: 1065_constexpr_invoke/constexpr-invoke.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ document: P1065R2
...
 date: today
 author:
-	- name: Tomasz Kamiński
-	  email: <tomaszkam@gmail.com>
-	- name: Barry Revzin
-	  email: <barry.revzin@gmail.com>
+  - name: Tomasz Kamiński
+    email: <tomaszkam@gmail.com>
+  - name: Barry Revzin
+    email: <barry.revzin@gmail.com>
```
Comment (mpark) [inline 1065_constexpr_invoke/constexpr-invoke.md]: Tabs are illegal characters in YAML. This and other Markdown files.

File: md/defaults.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import pathlib
+
+print(
+"""\
+css: {datadir}/pandoc.css
+filters: [{datadir}/pandoc.py]
+
+metadata:
+  highlighting:
+    inline-code: cpp\
```
Comment (mpark) [inline md/defaults.py]: This allows `code_cpp` to go away.

Comment (BRevzin) [reply md/defaults.py]: Is this a new pandoc thing or something that always existed and I just didn't know about?

**Comment by BRevzin** [comment]
Thanks for doing this!

I'm going to go ahead and flip all the Makefiles, since like:
```
brevzin@cpp-swim:~/linux/cpp_proposals/2089_param_constraints$ make
make: Circular param-constraints.pdf <- param-constraints.pdf dependency dropped.
make: Nothing to be done for 'all'.
```
but that's a trivial change on my end, just have to do:
```diff
diff --git a/2089_param_constraints/Makefile b/2089_param_constraints/Makefile
index 1a128ba..d4cd99c 100644
--- a/2089_param_constraints/Makefile
+++ b/2089_param_constraints/Makefile
@@ -1,3 +1,2 @@
-include ../md/mpark-wg21.mk
-
 d2089r0.html : param-constraints.md
+include ../md/mpark-wg21.mk
```
everywhere, and I can do that one.

File: md/defaults.py (hunk 2 above)
Comment (mpark) [reply md/defaults.py]: No, I implemented it yesterday: https://github.com/mpark/wg21/commit/f23a31ba796b88248003d4de809c6412648862e0

**Pull request merged**